docs/cli/ignite/ignite_create.md

```diff
@@ -14,8 +14,9 @@ the flags for this command.
 
 If the name flag (-n, --name) is not specified,
 the VM is given a random name. Using the copy files
-flag (-f, --copy-files), additional files can be added to
-the VM during creation with the syntax /host/path:/vm/path.
+flag (-f, --copy-files), additional files/directories
```

Comment:
`make tidy` added this?

Reply:
Yes, that's fine to include here, just forgot to update the docs in an earlier PR where I changed this :sweat_smile: